packages/documentation/src/stories/getting-started/compatibility/compatibility.docs.mdx

```diff
@@ -0,0 +1,95 @@
+import { Meta } from '@storybook/blocks';
+import * as GettingStartedStories from './compatibility.stories';
+
+<Meta of={GettingStartedStories} />
+
+# Compatibility
+
+<p id="compatibility-desc">List of compatible version numbers with external dependencies.</p>
+
+<table aria-describedby="compatibility-desc" className="table caption-top table-sm w-auto">
```

Comment:
Wrap all the tables with `<div class="table-responsive">.../div>`
And remove the classes `table-sm` and `w-auto` for a consisting look.